data/scripts/install-man-pages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/env sh
+
+# Set PREFIX.
+if [ -z  "${1+x}" ]; then
+    if ! [ -z "${PREFIX+x}" ]; then
+        PREFIX=/usr
+    fi
+else
+    PREFIX=${1}
+fi
+export PREFIX
+
+echo "Installing man pages"
+install -d ${PREFIX}/share/man/man1
```

Comment:
There's no need to invoke `install` to create both the directory and the file. This can be accomplished by simply using the `-D` flag.